test/Templates.Test/Helpers/WebDriverFactory.cs

```diff
@@ -24,13 +26,19 @@ public static IWebDriver CreateWebDriver()
             var result = IsAppVeyor ? CreateFirefoxDriver() : CreateEdgeDriver();
             result.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(1);
             return result;
+
+            IWebDriver CreateFirefoxDriver() =>
```

Comment:
Unreachable code (presumbly wasn't meant to be included in the PR)

Reply:
This gets called here https://github.com/aspnet/templating/pull/167/files/aef1328ed6ff41aeaace505f954798ce687acdab#diff-7765183c367cb5141c05166cc20ed8d7L24

Which in turn calls

https://github.com/aspnet/templating/pull/167/files/aef1328ed6ff41aeaace505f954798ce687acdab#diff-7765183c367cb5141c05166cc20ed8d7R40

Reply:
Oh I see. I misread it.

But it is really strange to declare this as an inline function like this, when there's a same-named function in the outer scope, which in turn no longer gets used except by this inline function.

To make it a bit more obvious, could you just change the existing `CreateFirefoxDriver` method so that it always sets `AcceptInsecureCertificates = true`, without needing the new local function?